### ISSUE TITLE
 Error: write ECONNRESET

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/bridge#338]
### Package Versions

Replace the values below using the output from `npm list storj-bridge`.

```
storj-bridge@4.3.2 /home/gordon/Code/storj-bridge
└── Storjcli: 4.0.0 | Core: 6.0.15

OS: windows 10
```

Replace the values below using the output from `node --version`.

```
v6.9.1
```

### Expected Behavior

Please describe the program's expected behavior. Include an example of your
usage code in the back ticks below if applicable.

```
Upload succesfull
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
[Sun Jan 22 2017 02:40:30 GMT+0100 (W. Europe Standard Time)] [info]   Audit generation for shard done.
 [Sun Jan 22 2017 02:40:30 GMT+0100 (W. Europe Standard Time)] [info]   Waiting on a storage offer from the network...
 [Sun Jan 22 2017 02:40:30 GMT+0100 (W. Europe Standard Time)] [info]   Querying bridge for contract for b7b0095e6c38128c833003244beaa89c53ee5214 (retry: 0)
 [Sun Jan 22 2017 02:40:32 GMT+0100 (W. Europe Standard Time)] [info]   Contract negotiated with: {"userAgent":"6.0.12","protocol":"1.0.0","address":"98.27.222.254","port"
:4000,"nodeID":"b692cf937543fcdfeb956274261feab3b8fa3c6f","lastSeen":1485049224138}
 [Sun Jan 22 2017 02:41:04 GMT+0100 (W. Europe Standard Time)] [warn]   Failed to transfer shard, reason: write ECONNRESET
 [Sun Jan 22 2017 02:41:04 GMT+0100 (W. Europe Standard Time)] [info]   Shard transfer failed 0 times, getting another contract...
 [Sun Jan 22 2017 02:41:04 GMT+0100 (W. Europe Standard Time)] [info]   Adding NodeID b692cf937543fcdfeb956274261feab3b8fa3c6f to blacklist
 [Sun Jan 22 2017 02:41:05 GMT+0100 (W. Europe Standard Time)] [info]   Hash for this shard is: b7b0095e6c38128c833003244beaa89c53ee5214
 [Sun Jan 22 2017 02:41:05 GMT+0100 (W. Europe Standard Time)] [info]   Waiting on a storage offer from the network...
 [Sun Jan 22 2017 02:41:05 GMT+0100 (W. Europe Standard Time)] [info]   Querying bridge for contract for b7b0095e6c38128c833003244beaa89c53ee5214 (retry: 0)
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: write ECONNRESET
    at exports._errnoException (util.js:1026:11)
    at WriteWrap.afterWrite (net.js:799:14)

Waiting for 0 seconds, press a key to continue ...
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. upload a file
2. ...
3. ...

**Comment by MeijeSibbel** [comment]
duplicate of: https://github.com/Storj/core-cli/issues/76
